wrapper/hosts.py

```diff
@@ -748,7 +748,6 @@ def prepare_command(self, data, v2v_args, v2v_env, v2v_caps):
                 v2v_args.extend(['-oo', 'rhv-verifypeer=%s' %
                                 ('false' if data['insecure_connection'] else
                                  'true')])
-
```

Comment:
some leftover here

Reply:
not sure I understand. this line is already deleted.